python/paddle/fluid/layer_helper.py

```diff
@@ -397,7 +398,7 @@ def append_activation(self, input_var):
         act = self.kwargs.get('act', None)
         if act is None:
             return input_var
-        if isinstance(act, basestring):
+        if isinstance(act, str):
```

Comment:
what if act is 'basestring' in py2? Would this line be silently skipped?

Reply:
some issue elsewhere.

Reply:
```basestring``` is not supported in py3 anymore, i will change it to ```six.string_types``` in next commit